rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2499,6 +2500,17 @@ public long getActiveFederationCreationBlockHeight() {
         return activeFederationCreationBlockHeightOpt.orElse(0L);
     }
 
+    public void checktimeLockExpiration() {
```

Comment:
this should probably be protected instead

Reply:
Done